src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../../utils/api";
+
+function SingleArticle(){
+    const[singleArticle, setSingleArticle]= useState({})
+ const {article_id}=useParams()
+ console.log(article_id);
+ useEffect(()=>{
```

Comment:
We also want to add a loading state, so if we are still awaiting the data a loading message can be displayed on the screen